robottelo/constants/__init__.py

```diff
@@ -316,7 +316,6 @@ class Colored(Box):
     'rhel7_extra': 'Red Hat Enterprise Linux 7 Server - Extras (RPMs)',
     'rhel7_optional': 'Red Hat Enterprise Linux 7 Server - Optional (RPMs)',
     'rhel7_sup': 'Red Hat Enterprise Linux 7 Server - Supplementary (RPMs)',
-    'rhst7_610': 'Red Hat Satellite Tools 6.10 (for RHEL 7 Server) (RPMs)',
```

Comment:
is this intended change? there are few occurances of this constant in a few tests, so it could cause failures

Reply:
This is a duplicate entry that Ruff detected. Therefore ruff automatically deleted it when I used `--fix`.